todo_app/data/session_items.py

```diff
@@ -1,41 +1,30 @@
+import os
 from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+from todo_app.data.api_client import get_cards_request, create_new_card, move_card_to_list
+from todo_app.data.card import Card
 
 def get_items():
-    return session.get('items', _DEFAULT_ITEMS.copy())
-
-def update_task_status(id):
-    item = get_item(id)
-    if (item["status"] == "Not Started"):
-        item["status"] = "Complete"
-    else:
-        item["status"] = "Not Started"
-    save_item(item)
-
-def get_item(id):
-    items = get_items()
-    return next((item for item in items if item['id'] == int(id)), None)
-
-def add_item(title):
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-    items.append(item)
-    session['items'] = items
-
-    return item
-
-def save_item(item):
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-    session['items'] = updated_items
-
-    return item
+    cards_result = get_cards_request()
+    card_array = []
+    for list in cards_result:
+            for card in list["cards"]:
+                card_array.append(
+                    Card.from_trello_card(card, list)
+                )
```

Comment:
Not necessarily a suggestion, but worth a comparison! We can take a more functional approach to building this array; do you find it cleaner or a bit too concise and unclear?

```python
    card_array = [
        Card.from_trello_card(card, list)
        for list in cards_result
        for card in list['cards']
    ]
```